data/strings/sound.txt

```diff
@@ -304,6 +320,21 @@
     zh-Hans = 向左急转。
     zh-Hant = 向左急轉。
 
+  [make_a_left_turn_street]
```

Comment:
Does it make sense to embed this translation into dist_direction_onto_street instead of defining it separately?